pkg/chunkenc/interface.go

```diff
@@ -111,6 +111,7 @@ type Chunk interface {
 	SampleIterator(ctx context.Context, from, through time.Time, extractor log.StreamSampleExtractor) iter.SampleIterator
 	// Returns the list of blocks in the chunks.
 	Blocks(mintT, maxtT time.Time) []Block
+	// Size returns the number of entries in a chunk
```

Comment:
I'm confused by the comment ? Should it be the size of the chunk ?

Reply:
It _should_, but we've historically used it as the number of entries:
https://github.com/grafana/loki/blob/main/pkg/ingester/stream.go#L140-L143
https://github.com/grafana/loki/blob/main/pkg/ingester/flush.go#L398

Reply:
Should you fix the comment ? That's where I'm confused.

Reply:
I think the comment is fixed - we use `chunkenc.Chunk`'s `Size()` method to give the number of entries. This is different from Cortex's `Chunk` type, which the `Facade` now correctly accounts for. 

I can also change the method name, but I figured this was the least intrusive change.